src/aux/aux_line_like.cpp

```diff
@@ -240,4 +215,4 @@ bool line_like(string model_line, vector<string> &tokens, compiler_state &state)
     if (j < tokens.size())
```

Comment:
I've been checking this error and it's quite weird, but I think I got it. The thing is: the way our check was before your pr is that if the tokens DID NOT match what they were expected to match, they would fall into this last test and then decide if they should return true or false. In your modified version, however, in most cases you don't reach this test because the match has already returned. Sorry 😢